designer/server/src/main/scala/pl/touk/nussknacker/ui/util/PdfExporter.scala

```diff
@@ -204,37 +204,39 @@ object PdfExporter extends LazyLogging {
   </block>
 
   private def nodeDetails(node: NodeData) = {
-    val nodeData = node match {
-      case Source(_, SourceRef(typ, params), _) => ("Type", typ) :: params.map(p => (p.name, p.expression.expression))
-      case Filter(_, expression, _, _)          => List(("Expression", expression.expression))
+    val nodeData: List[(String, String)] = node match {
+      case Source(_, SourceRef(typ, params), _) =>
+        ("Type", typ) :: params.map(p => (p.name.value, p.expression.expression))
+      case Filter(_, expression, _, _) => List(("Expression", expression.expression))
       case Enricher(_, ServiceRef(typ, params), output, _) =>
-        ("Type", typ) :: ("Output", output) :: params.map(p => (p.name, p.expression.expression))
+        ("Type", typ) :: ("Output", output) :: params.map(p => (p.name.value, p.expression.expression))
       // TODO: what about Swtich??
       case Switch(_, expression, exprVal, _) => expression.map(e => ("Expression", e.expression)).toList
       case Processor(_, ServiceRef(typ, params), _, _) =>
-        ("Type", typ) :: params.map(p => (p.name, p.expression.expression))
-      case Sink(_, SinkRef(typ, params), _, _, _) => ("Type", typ) :: params.map(p => (p.name, p.expression.expression))
+        ("Type", typ) :: params.map(p => (p.name.value, p.expression.expression))
+      case Sink(_, SinkRef(typ, params), _, _, _) =>
+        ("Type", typ) :: params.map(p => (p.name.value, p.expression.expression))
       case CustomNode(_, output, typ, params, _) =>
-        ("Type", typ) :: ("Output", output.getOrElse("")) :: params.map(p => (p.name, p.expression.expression))
+        ("Type", typ) :: ("Output", output.getOrElse("")) :: params.map(p => (p.name.value, p.expression.expression))
       case FragmentInput(_, FragmentRef(typ, params, _), _, _, _) =>
-        ("Type", typ) :: params.map(p => (p.name, p.expression.expression))
-      case FragmentInputDefinition(_, parameters, _) => parameters.map(p => p.name -> p.typ.refClazzName)
+        ("Type", typ) :: params.map(p => (p.name.value, p.expression.expression))
+      case FragmentInputDefinition(_, parameters, _) => parameters.map(p => p.name.value -> p.typ.refClazzName)
       case FragmentOutputDefinition(_, outputName, fields, _) =>
         ("Output name", outputName) :: fields.map(p => p.name -> p.expression.expression)
       case Variable(_, name, expr, _) => (name -> expr.expression) :: Nil
       case VariableBuilder(_, name, fields, _) =>
         ("Variable name", name) :: fields.map(p => p.name -> p.expression.expression)
       case Join(_, output, typ, parameters, branch, _) =>
         ("Type", typ) :: ("Output", output.getOrElse("")) ::
-          parameters.map(p => p.name -> p.expression.expression) ++ branch.flatMap(bp =>
+          parameters.map(p => p.name.value -> p.expression.expression) ++ branch.flatMap(bp =>
             bp.parameters.map(p => s"${bp.branchId} - ${p.name}" -> p.expression.expression)
```

Comment:
```suggestion
            bp.parameters.map(p => s"${bp.branchId} - ${p.name.value}" -> p.expression.expression)
```
last one - type check passes with List[(String, String)] right now since its inside a template

Reply:
Good catch!